report-job-status/action.yml

```diff
@@ -32,6 +39,19 @@ runs:
           ${{ github.event.head_commit.message }}
       shell: bash
 
+    - name: Get job data
+      id: get-job-data
+      shell: bash
+      env:
+        CURL_COMMAND: "curl --location --fail --silent --show-error --retry 5 --retry-delay 5 "
+        API_ROOT: https://api.github.com/repos/${{ github.repository }}/actions/runs
+      run: |
+        echo 'Fetching job data'
+        echo 'JOB_CONTEXT<<EOF' >> $GITHUB_ENV
+        ${{ env.CURL_COMMAND }} -H 'authorization: Bearer ${{ github.token }}' \
+        ${{ env.API_ROOT }}/${{ github.run_id }}/jobs >> $GITHUB_ENV
```

Comment:
Can it be moved to the right for more readability?

Reply:
fixed